packages/image_picker/image_picker_android/README.md

```diff
@@ -7,5 +9,17 @@ The Android implementation of [`image_picker`][1].
 This package is [endorsed][2], which means you can simply use `image_picker`
 normally. This package will be automatically included in your app when you do.
 
+## Photo Picker
+
+This package has optional Android Photo Picker functionality.
```

Comment:
Let's make "Android Photo Picker" a link to https://developer.android.com/training/data-storage/shared/photopicker since people may not know what this means.